lib/invoiceCollection.js

```diff
@@ -0,0 +1,41 @@
+invoices = new Mongo.Collection("invoices");
+invoices.byTimeRange = function(timeRange = "today", queryParams) {
+        let limit = 100;
+        if(queryParams.hasOwnProperty('limit')) {
+            limit = parseInt(queryParams.limit);
+            delete queryParams['limit'];
+        }
+        console.log('limit', limit);
+
+        queryParams = (isEmpty(queryParams))?{sortCreatedAt: "DESC", sortTotal: "ASC"}:queryParams;
+
+        const sort = {};
+        for (let k in queryParams) {
+            sort[k.replace('sort', '').firstToType('lower')] = (queryParams[k] === 'DESC')?-1:1;
+        }
```

Comment:
please improve for readability, it takes too long to parse what's going on here. Maybe also add some inline comments on the way. Remember code is read 80% of the time, so it should be extremely easy to understand!